src/FSharpPlus/Lens.fs

```diff
@@ -113,19 +113,43 @@ module Lens =
     let inline _5 f t = Map.InvokeOnInstance (fun x -> mapItem5 (fun _ -> x) t) (f (item5 t))
 
     [<RequireQualifiedAccess>]
-    module Set=
+    module List =      
+        /// Given a specific key, produces a Lens from a List<value> to an Option<value>. When setting,
+        /// a Some(value) will insert or replace the value into the map at the given index. Setting a value of
```

Comment:
"into the map at the given index." -> "into the **list** at the given index."

Reply:
Copy-pasting bites again 😅